### PR TITLE
Expand for Monnit IoT Sensors

### PR DESCRIPTION
**Pull request by cstirdivant**
Expanding subfields for predictive maintenance IoT pilot.

Expanding units of measurement for gas meters.

@tasodorff @mschulze17

### CODE REVIEW

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ component:
   pump: "Component used for the distribution of liquid media."
   riser: "Water riser for heating, cooling and so on"
   shade: "Window covering."
+  signal: "An electric current or electromagnetic field used to convey data from one place to another."
```
Comment (tasodorff) [inline ontology/yaml/resources/subfields/subfields.yaml:48]: is this really a `component`? this seems like a `descriptor` to me.

**Review by tasodorff — COMMENTED** [review]
Not sure on `signal` as a `component`. Please explain.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 1 above)
Comment (cstirdivant) [reply ontology/yaml/resources/subfields/subfields.yaml]: ack this should be a descriptor

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -1039,3 +1039,13 @@ literals:
...
 - local_air_temperature_sensor
 
+## New fields for IoT Vibration devices
+- tenminutefixed_max_xaxis_linearvelocity_sensor
+- tenminutefixed_max_yaxis_linearvelocity_sensor
+- tenminutefixed_max_zaxis_linearvelocity_sensor
+- tenminutefixed_max_xaxis_frequency_sensor
+- tenminutefixed_max_yaxis_frequency_sensor
+- tenminutefixed_max_zaxis_frequency_sensor
+- signal_percentage_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1049]: `signal_strength_percentage_sensor`?

**Review by tasodorff — CHANGES_REQUESTED** [review]
Can you also confirm if the units of measurement for the linearvelocity sensor are `millimeters_per_second`, and that those exist?

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 2 above)
Comment (cstirdivant) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: updated to signal_strength_percentage_sensor

**Comment by cstirdivant** [comment]
> Can you also confirm if the units of measurement for the linearvelocity sensor are `millimeters_per_second`, and that those exist?

Con

**Review by tasodorff — APPROVED** [review]

**Comment by cstirdivant** [comment]
@tasodorff please review the new HVAC/SENSOR type

**Review by charbull — APPROVED** [review]

**Pull request merged**